appveyor.yml

```diff
@@ -27,7 +27,7 @@ install:
 build_script:
 # Need to convert from shallow to complete for Pkg.clone to work
   - IF EXIST .git\shallow (git fetch --unshallow)
-  - C:\projects\julia\bin\julia -e "versioninfo(); Pkg.init(); Pkg.add(\"LibCURL\"); Pkg.checkout(\"LibCURL\"); Pkg.clone(\"https://github.com/samuel-massinon-invenia/JavaCall.jl.git\"); Pkg.checkout(\"JavaCall\", \"pull-request/bf8b4987\"); Pkg.clone(pwd(), \"FTPClient\"); Pkg.build(\"FTPClient\");"
+  - C:\projects\julia\bin\julia -e "versioninfo(); Pkg.init(); Pkg.add(\"LibCURL\"); Pkg.checkout(\"LibCURL\"); Pkg.clone(pwd(), \"FTPClient\"); Pkg.build(\"FTPClient\");"
```

Comment:
Totally not your fault but adding LibCURL should be handled by the REQUIRE file. Remove the section:
```
Pkg.init(); Pkg.add(\"LibCURL\"); Pkg.checkout(\"LibCURL\"); 
```
You'll also want to make this change to the .travis.yml file. Make this in a separate commit than the change above.